akka-docs/rst/scala/remoting-artery.rst

```diff
@@ -281,11 +281,56 @@ untrusted mode when incoming via the remoting layer:
   marking them :class:`PossiblyHarmful` so that a client cannot forge them.
 
 
-Lifecycle and Failure Recovery Model
-------------------------------------
-
-TODO
-
+Quarantine
+----------
+
+Akka remoting is using Aeron as underlying message transport. Aeron is using UDP and adds
+among other things reliable delivery and session semantics, very similar to TCP. This means that
+the order of the messages are preserved, which is needed for the :ref:`Actor message ordering guarantees <message-ordering>`.
+Under normal circumstances all messages will be delivered but there are cases when messages
+may not be delivered to the destination:
+
+* during a network partition and the Aeron session is broken
+* when sending to many messages without flow control and thereby filling up the outbound send queue (``outbound-message-queue-size`` config)
+* if serialization or deserialization of a message fails (only that message will be dropped) 
+* if an unexpected exception occurs in the remoting infrastructure
+
+In short, Actor message delivery is “at-most-once” as described in :ref:`message-delivery-reliability`
+
+Some messages in Akka are called system messages and those cannot be dropped because that would result
+in an inconsistent state between the systems. Such messages are used for essentially two features; remote death
+watch and remote deployment. These messages are delivered by Akka remoting with “exactly-once” guarantee by
+confirming each message and resending unconfirmed messages. If a system message anyway cannot be delivered the 
+association with the destination system is irrecoverable failed. It is placed in a so called quarantined state. 
```

Comment:
Mention also that quarantine usually does not happen if no remote watch or remote deployment is used.